modules/internal/ChapelDistribution.chpl

```diff
@@ -353,6 +353,9 @@ module ChapelDistribution {
 
     pragma "last resort" @chpldoc.nodoc
     proc dsiCreateIndexBuffer(size) { dnsError("createIndexBuffer"); }
+    
+    pragma "last resort" @chpldoc.nodoc
+    proc dsiCreateIndexBuffer(size,dataSorted=false,isUnique=false) { dnsError("createIndexBuffer"); }
```

Comment:
Did you really need to add a new overload of `dsiCreateIndexBuffer`? You could just add the default-false arguments to the existing function, right? If that's the case can you make that change?

Reply:
I did try to add the default-false arguments to the existing function, but the compiler yelled at me due to the procedures that `override` this base one not having the same signature. 

Therefore, instead of going around and changing where `dsiCreateIndexBuffer` is overridden, I created a new function. However, if you think that changing every `override proc dsiCreateIndexBuffer(size)` to use the new flags is the correct and proper way to handle this change, then I will do it!